admin-core/src/main/java/com/depromeet/domain/franchise/infrastructure/persistence/FranchiseRepositoryImpl.java

```diff
@@ -26,19 +27,27 @@ public List<Franchise> findAll() {
 
     @Override
     public void save(final Franchise franchise) {
-        final FranchiseEntity franchiseEntity = Franchise.toEntity(franchise);
+        final var franchiseEntity = Franchise.toEntity(franchise);
```

Comment:
정적 메서드를 사용하지 않아도 충분히 원하고자 하는 로직을 할 수 있을 것 같아요 :smile
왜냐하면 파라미터로 들어온 franchise 도메인 객체는 이미 entity을 구성하는데 필요한 데이터를 알고 있어서요~

```suggestion
        final FranchiseEntity franchiseEntity = franchise.toEntity();
```